modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -44,4 +44,19 @@ public static class Table {
         /** Column not found. */
         public static int COLUMN_NOT_FOUND_ERR = TABLE_ERR_GROUP.registerErrorCode(4);
     }
+
+    /** Replicator error group. */
+    public static class Replicator {
+        /** Replicator error group. */
+        public static final ErrorGroup REPLICATOR_ERR_GROUP = ErrorGroup.newGroup("REP", 3);
```

Comment:
Could you please merge fresh ignite3_tx? Denis already added group with code 3.
`        public static final ErrorGroup TX_ERR_GROUP = ErrorGroup.newGroup("TX", 3);`